src/pages/signup.tsx

```diff
@@ -0,0 +1,50 @@
+import type { NextPage } from 'next';
+import Header from '../components/header'
+import Link from 'next/link'
+
+const SignUp: NextPage = () => {
+    return (
+        <>
+            <Header />
+            <section className="h-screen flex items-center bg-gradient-to-r from-complementary to-dominant">
```

Comment:
Similar to the login page, we should have tests to make sure the `create an account` button doesn't work without all fields entered.

Again, the API call could be checked too.

Reply:
As discussed, we won't be worrying about testing on the frontend due to time constraints.